src/opm/input/eclipse/EclipseState/Runspec.cpp

```diff
@@ -270,28 +270,29 @@ bool WellSegmentDims::operator==(const WellSegmentDims& data) const
         && (this->location() == data.location());
 }
 
-NetworkDims::NetworkDims() :
-    nMaxNoNodes( 0 ),
-    nMaxNoBranches( 0 ),
-    nMaxNoBranchesConToNode( ParserKeywords::NETWORK::NBCMAX::defaultValue )
+NetworkDims::NetworkDims()
+    : nMaxNoNodes(0)
+    , nMaxNoBranches(0)
+    , nMaxNoBranchesConToNode(ParserKeywords::NETWORK::NBCMAX::defaultValue)
 {}
 
-NetworkDims::NetworkDims(const Deck& deck) : NetworkDims()
+NetworkDims::NetworkDims(const Deck& deck)
```

Comment:
While here, make this explicit.

Reply:
> While here, make this explicit.

I don't quite follow this comment.  The constructor is already marked `explicit` [in the header file](https://github.com/OPM/opm-common/blob/ed18e31e913a2aeee0b4dcef135254e61ae2e91c/opm/input/eclipse/EclipseState/Runspec.hpp#L186).

Reply:
Sorry, ignore that.